### PR TITLE
`X-Prerender-Token` is not returned for ordinary requests anymore. Fixes #5

### LINKED ISSUE #5: Initial request response headers contain the `X-Prerender-Token`

### PR DESCRIPTION
**Pull request by EECOLOR**

### CODE REVIEW

**Comment by klaasman** [comment]
in the [README.md](https://github.com/Kaliber/play-prerender/blob/master/README.md): 

```
# Token for prerender.io. Will be shown in X-Prerender-Token response header on the index page.
```

File: .settings/org.scala-ide.sdt.core.prefs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#Generated by sbteclipse
```
Comment (klaasman) [inline .settings/org.scala-ide.sdt.core.prefs]: is this file necessary?

**Comment by EECOLOR** [comment]
@klaasman :+1:

**Comment by Hayena** [comment]
:+1:
